net/tailscale/files/tailscale.init

```diff
@@ -43,5 +47,5 @@ start_service() {
 }
 
 stop_service() {
-  /usr/sbin/tailscaled --cleanup
+  /usr/sbin/tailscaled --cleanup --no-logs-no-support
```

Comment:
Again, this is placed in several different locations. Isn’t it sufficient to have it in just one place?